keanu-project/src/test/java/io/improbable/keanu/algorithms/variational/KLDivergenceTest.java

```diff
@@ -0,0 +1,31 @@
+package io.improbable.keanu.algorithms.variational;
+
+import io.improbable.keanu.algorithms.NetworkSamples;
+import io.improbable.keanu.algorithms.mcmc.MetropolisHastings;
+import io.improbable.keanu.network.BayesianNetwork;
+import io.improbable.keanu.vertices.ProposalVertex;
+import io.improbable.keanu.vertices.dbl.nonprobabilistic.ConstantDoubleVertex;
+import io.improbable.keanu.vertices.dbl.nonprobabilistic.operators.binary.AdditionVertex;
+import io.improbable.keanu.vertices.dbl.probabilistic.GaussianVertex;
+import static org.junit.Assert.assertTrue;
+import org.junit.Test;
+
+import java.util.Collections;
+
+public class KLDivergenceTest {
+    @Test
+    public void testKLDivergence() {
```

Comment:
JUnit isn't great at helping developers describe the intention of their test but ideally the method name is the test name and the test name should describe what the assertion in the test is. This also encourages single assertions in tests which is good practice.